src/components/sidebar/networkView/nodeView/NodeView.tsx

```diff
@@ -98,8 +99,7 @@ class NodeView extends ViewFormBase<INodeViewProps, INodeViewState> {
             this.props.nodeStore!.setCurrentStateAsOld();
             this.props.dialogStore!.setFadeMessage('Tallennettu!');
         } catch (err) {
-            const errMessage = err.message ? `, (${err.message})` : '';
-            this.props.errorStore!.addError(`Tallennus epäonnistui${errMessage}`);
+            showError(err);
```

Comment:
What was wrong with how it was before?

Reply:
As in #662, I would add error param to addMessage

```
this.props.errorStore!.addError(Tallennus epäonnistui., error);

@action
public addMessage(message: string, error?: Error) {
    let msg = message;
    const statusCode = error.errorCode;
    if(error.errorCode) {
           msg += ' ' + httpStatusCodeList[error.errorCode];
     }
    this._errors.push(msg);
    /* tslint:disable no-console */
    if(error) console.error(error);
    /* tslint:enable no-console */
}
```

Reply:
Refactor to using logic more similar to this example